channels_jsonrpc/jsonrpcconsumer.py

```diff
@@ -231,10 +243,11 @@ def raw_receive(self, message, **kwargs):
         :return:
         """
         content = '' if "text" not in message else message["text"]
-        result = self.__handle(content, message)
+        result, result_type = self.__handle(content, message)
 
-        # Send responce back
-        self.send(text=self._encode(result))
+        # Send responce back only if it is a call, not notification
+        if result_type == 0:
```

Comment:
@soulhunter1987 
please use some GLOBAL_VAR for the result_type:
RESULT_NOTIFICATION_TYPE/RESULT_REQUEST_TYPE, for clarity purposes, or 'is_notification as variable name' ...